tests/integration/utils.go

```diff
@@ -438,6 +442,42 @@ func (tk *TestKeeper) PGDataVersion() (int, int, error) {
 	return pg.ParseVersion(version)
 }
 
+// GetPGControldata calls pg_controldata for the Postgres database directory. This can be
+// useful in debugging database state, especially when comparing what was successfully
+// checkpointed.
+func (tk *TestKeeper) GetPGControldata() string {
```

Comment:
Unused but was useful while debugging. Can remove if we want, or leave for future use.